README.md

```diff
@@ -129,6 +130,26 @@ is not supported by `buf-push-action` on its own - you'll need to stitch this fu
 your workflow on your own. For more details, see [this](https://github.com/bufbuild/buf/issues/838)
 discussion.
 
+### Push with a custom tag
```

Comment:
Currently, you cannot `buf push` with both a `--draft` and a `--tag`. This should be clearly called out in the documentation.

Suggestion: at the end of this section, I would add the following warning:

```
Important note: you cannot set both `--tag` and `--draft`. You can only use one or the other.
```